### DISCUSSION TITLE
How does editing work exactly? What happens after an edit is submitted?

### DISCUSSION THREAD

**Discussion by squeak9** [organicmaps/organicmaps#1349] [Q&A] [answered]
I am already editing OSM through other apps like iD, JOSM, OSMAnd, StreetComplete, etc but this is app confusing me..

If I'm not logged in and send an edit then where does it go? Does it just become a [standard note on the osm.org map](https://wiki.openstreetmap.org/wiki/Notes)? There is a popup warning when you're about to submit a change that says:
> _"do you want to send it to all users?"_ 

which suggests it's a note but it also says:
> _"if we have any questions **we will contact you by email**"_

 which is very confusing since the app doesn't have my email..

If you are logged in does the edit get added to osm directly via my account? the popup doesn't change if you are logged in.

This should be clarified somewhere, probably in the app, maybe in the github wiki or just in this discussion.

**Comment by markcellus** [comment]
Agree. 

Also, adding a new address on Android doesn't give you all the options like on the OSM website (e.g suite/unit numbers).

**Comment by biodranik** [comment, marked as answer]
The wording should be fixed. Without logging in, your edits are not uploaded, only saved locally.
Reporting places as non-existent however uploads a note.

Contacting by email can happen only for logged users, when someone comments on a note.
